db/seeds.rb

```diff
@@ -5,3 +5,11 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+
+# before running "rake db:seed", do the following:
+# - put the "rails-engine-development.pgdump" file in db/data/
+cmd = "pg_restore --verbose --clean --no-acl --no-owner -h localhost -U $(whoami) -d rails_engine_ii_development db/data/rails-engine-development.pgdump"
```

Comment:
It was necessary to change the provided script to match the project name. Possibly note this in the documentation, if appropriate.